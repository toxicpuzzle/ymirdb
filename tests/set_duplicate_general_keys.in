SET a 1 2
SET b a a 2 2
LEN b
LEN a
LIST ENTRIES
bye