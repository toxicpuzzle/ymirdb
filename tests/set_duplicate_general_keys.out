> ok

> ok

> 6

> 2

> b [a a 2 2]
a [1 2]

> bye